src/Components/RecommendedSection/RecommendedSection.tsx

```diff
@@ -0,0 +1,64 @@
+import { Button } from '@chakra-ui/button';
+import { Box, Heading, SimpleGrid } from '@chakra-ui/layout';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import { btnStyleProps, solidPrimaryButtonStyleProps } from '../utils';
+import { Video } from './Video.type';
+import { VideoCard } from './VideoCard';
+
+export const RecommendedSection = ({ type }: { type: string }) => {
+	const [videos, setVideos] = useState<Video[] | null>(null);
+
+	const gridProps = {
+		columns: [1, 2, 3],
+		gap: '1.5rem',
+		px: '1rem',
+		maxW: '67rem',
+		margin: 'auto',
+	};
+
+	useEffect(() => {
+		(async () => {
+			try {
+				const url = `https://uandistoreapi.herokuapp.com/videos/type/${type}`;
```

Comment:
You could replace the URL with API_URL which is already defined in constants.js
Further, rename this file to TypeScript instead of JavaScript.

Reply:
Thank you for pointing it out, I somehow missed it.